README.rst

```diff
@@ -55,3 +55,25 @@ functions.
 Development
 ***************
 We follow this `git branching model <http://nvie.com/posts/a-successful-git-branching-model/>`_, please have a look at it.
+
+
+*******
+Install Instructions (from pip's database)
+*******
+pip install pybloom-live
+
+*******
+Install Instructions (Windows build-from-GitHub source)
+*******
+  1) Download the zip file by clicking on the green "Clone or Download" link followed by "Download zip"
+  2) Extract all the contents of the the zip folder
+  3) Open command prompt (cmd.exe) to the extracted folder.
+    a) Find the extracted folder in Windows Explorer.
+    b) From the parent-level Shift+RightClick on the folder.
+    c) Select "open command window here".
+  4) type `pip install .`
```

Comment:
Capitalise the first word of the sentence and use full stop consistently. 

For an example:

  1. Download the zip file by clicking on the green "Clone or Download" link followed by "Download zip".
  2. Extract all the contents of the the zip folder.
  3. Open command prompt (cmd.exe) to the extracted folder.
      * Find the extracted folder in Windows Explorer.
      * From the parent-level Shift+RightClick on the folder.
      * Select "open command window here".
  4. Type `pip install .`